runtime/executor/src/test/java/org/apache/nemo/runtime/executor/data/BlockStoreTest.java

```diff
@@ -229,7 +229,7 @@ public void setUp() throws Exception {
   public void testMemoryStore() throws Exception {
     final Injector injector = Tang.Factory.getTang().newInjector();
     injector.bindVolatileInstance(SerializerManager.class, serializerManager);
-    injector.bindVolatileParameter(JobConf.MaxOffheapMb.class, 128);
+    injector.bindVolatileParameter(JobConf.ExecutorMemoryMb.class, 640);
```

Comment:
I think it would be more readable if we specify the `MaxOffheapRatio` to `0.2`. Others may not be aware of the default value.

Reply:
I have explicitly binded MaxOffheapRatio with default value 0.2.